DOCS-577 PGD-5.5-Release (#5570)
EnterpriseDB/docs: product_docs/docs/pgd/4/rel_notes/pgd_4.3.5_rel_notes.mdx

---
title: "Release notes for EDB Postgres Distributed version 4.3.5"
navTitle: "Version 4.3.5"
redirects:
  - /pgd/latest/bdr/release_notes/bdr4.3.5_rel_notes/
---

Released: 14 May 2024

EDB Postgres Distributed version 4.3.5 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.

!!! Note
This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.10, 15.5 and later.
!!!

| Component | Version | Type         | Description                                                                                        |
|-----------|---------|--------------|----------------------------------------------------------------------------------------------------|
| HARP      | 2.4.1   | Change       | Routine security library upgrades.                                                                 |

> | BDR       | 5.5.0   | Enhancement  | Log completion of an extension upgrade.                                                            |

Reviewer: @djw-m all of these say 5.4. when they should say 4.3.5

Author: Fixed

> | BDR       | 5.5.0   | Enhancement  | Log completion of an extension upgrade.                                                            |

Reviewer: @djw-m did we get rid of the support tickets column? thought that was helpful for customers to reference - Mastercard will be looking for this attach/detach fix

> | BDR       | 4.3.5   | Enhancement | Log completion of an extension upgrade.                                                            |           |

Reviewer 2: ```suggestion
| BDR       | 4.3.5   | Enhancement | BDR now completes logs of an extension upgrade.                                                           |           |
```

| BDR       | 4.3.5   | Enhancement | Execute each autopartition task in its own transaction.                                            |           |
| BDR       | 4.3.5   | Enhancement | Use `DETACH CONCURRENTLY` to drop partitions.                                                      | RT101407/35476  |
| BDR       | 4.3.5   | Enhancement | Grant usage of `pg_file_settings` and `pg_show_all_file_settings()` to bdr_superuser.              |           |

> | BDR       | 4.3.5   | Enhancement | Stop the initial replication connection for `bdr_init_physical` and start it only when needed.     | RT102828/35305  |

Reviewer 2: ```suggestion
| BDR       | 4.3.5   | Enhancement | BDR now stops the initial replication connection for `bdr_init_physical` and starts it only when needed.     | RT102828/35305  |
```

> | BDR       | 4.3.5   | Bug Fix     | Improve handling of node group configuration parameter "check_constraints".                        | RT99956/31896   |

Reviewer 2: ```suggestion
| BDR       | 4.3.5   | Bug Fix     | Improved handling of node group configuration parameter `check_constraints`.                        | RT99956/31896   |
```

> | BDR       | 4.3.5   | Bug Fix     | Prevent potential segfault in `bdr.monitor_group_versions()`.                                      | RT102290/34051  |

Reviewer 2: ```suggestion
| BDR       | 4.3.5   | Bug Fix     | Fixed an issue to prevent potential segfault in `bdr.monitor_group_versions()`.                                      | RT102290/34051  |
```

> | BDR       | 4.3.5   | Bug Fix     | Use bdr.default_sequence_kind when updating sequence kind of existing sequences upon node creation.|           |

Reviewer 2: ```suggestion
| BDR       | 4.3.5   | Bug Fix     | BDR now use `bdr.default_sequence_kind` when updating sequence kind of existing sequences upon node creation.|           |
```

---
title: "Release notes for EDB Postgres Distributed version 4.3.5"
navTitle: "Version 4.3.5"
redirects:
  - /pgd/latest/bdr/release_notes/bdr4.3.5_rel_notes/
---

Released: 16 May 2024

EDB Postgres Distributed version 4.3.5 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.

!!! Note
This version is required for EDB Postgres Advanced Server versions 12.15, 13.11, 14.10, 15.5, and later.
!!!

> | Component | Version | Type        | Description                                                                                                    | Addresses      |

Author: Because the column is for support tickets and CVEs. And also because "Related Support Tickets" is very wide and would push the actual description into around 25 characters wide. I can change it to "Ticket/CVE" which isn't pretty but would denote what the column is.

> | Component | Version | Type        | Description                                                                                                    | Addresses      |

Author: Changed to "Ticket"
